packages/integration-tests/src/test-e2e/create_amplify.test.ts

```diff
@@ -32,11 +32,18 @@ void describe(
       // Force 'create-amplify' installation in npx cache by executing help command
       // before tests run. Otherwise, installing 'create-amplify' concurrently
       // may lead to race conditions and corrupted npx cache.
-      await execa('npm', ['create', amplifyAtTag, '--yes', '--', '--help'], {
-        // Command must run outside of 'amplify-backend' workspace.
-        cwd: os.homedir(),
-        stdio: 'inherit',
-      });
+      const output = await execa(
+        'npm',
+        ['create', amplifyAtTag, '--yes', '--', '--help'],
+        {
+          // Command must run outside of 'amplify-backend' workspace.
+          cwd: os.homedir(),
+        }
+      );
+      const helpMessage =
```

Comment:
It would be more readable to check that the message contains a few strings that we care about to sanity check that it worked. Similar to what we do [here](https://github.com/aws-amplify/amplify-backend/blob/a3447c85426c0af2dfb714f24586b92cba0a425e/packages/cli/src/commands/sandbox/sandbox_command.test.ts#L109-L114)